lib/taiko.js

```diff
@@ -3159,48 +3159,6 @@ const rectangle = async (selector, callback) => {
  *
  */
 
-/**
```

Comment:
These are properties of ElementWrapper, and the existing ElementWrapper definition has to be updated to Element. It has to be updated and not removed.

Reply:
Pushing corrections. Thanks for the pointer

I also fixed the types definitions the best I could by adding methods from different Elements, in the same manner that it was done with ElementWrapper